README.md

```diff
@@ -17,4 +17,12 @@ python run_test.py dev apiKey ../files/
 
 The first argument to the script should be the environment that the script should run on, the second should be the `apiKey` for that environment, and the third the directory containing files for upload.
 
+In  powershell, run these commands (reference : https://www.c-sharpcorner.com/article/steps-to-set-up-a-virtual-environment-for-python-development/)
+```
+python3 -m venv venv
+.\venv\Scripts\activate
```

Comment:
This is the only different command, so I think it's worthwhile focusing on just this line.  You could put it above as a comment, like, "On Windows / in Powershell, use .\venv\Scripts\activate"

Thanks much for adding this and especially pointing to the article!

Reply:
Or you could keep it here but just say, "In Powershell, use `.\venv\Scripts\activate` instead of `source venv/bin/activate`"